src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -451,12 +451,347 @@ The following changes must be performed:
    ...
 ----
 
+=== Deploy ProActive Nodes dynamically via other schedulers (PBS, SLURM, ...)
+
+This functionality is also called `Meta-Scheduling`.
+
+If an existing cluster is available in your organization and this cluster is managed by a native scheduler such as *SLURM*, *LSF* or *PBS*, you may want to execute transparently on this cluster ProActive <<__glossary_workflow,*Workflows*>> and <<_glossary_task,*Tasks*>>.
+
+As ProActive Tasks can be executed only on <<_glossary_proactive_node,*ProActive Nodes*>>, it is necessary to deploy ProActive Nodes on the cluster.
+
+Dynamic deployment of ProActive Nodes on a cluster to execute ProActive Tasks is handled by defining a _Native Scheduler <<_glossary_node_source,Node Source>>_ containing three components:
+
+ * The `Native Scheduler Infrastructure`: a <<_glossary_node_source_infrastructure,*Node Source Infrastructure*>> which allows to interact with a native scheduler to deploy ProActive Nodes.
+
+ * The `Native Scheduler Policy`:  a <<_glossary_node_source_policy,*Node Source Policy*>> which interacts with the Native Scheduler Infrastructure to request ProActive Nodes deployment dynamically based on the <<_glossary_proactive_scheduler,*ProActive Scheduler*>> pending queue.
+
+ * The `Native Scheduler Scheduling Policy`: a <<_glossary_scheduling_policy,*Scheduling Policy*>> which allows the execution of ProActive Tasks on ProActive Nodes provisioned by the Native Scheduler Policy.
+
+The provisioning of ProActive Nodes is controlled by a specific <<../user/ProActiveUserGuide.adoc#_generic_information,*Generic Information*>> `"NS"` which can be defined at Task or Job level.
+
+When this generic information is configured for a _Task_, a ProActive Node will be dynamically deployed on the cluster to execute the Task. This ProActive Node will be associated with the Task and will only accept this Task execution. When the Task terminates, the ProActive Node will also be terminated.
+
+When this generic information is configured for a _Job_, all Tasks contained in the Job will execute on the cluster. Nodes will be created dynamically to execute Tasks of this Job, with a similar Task/Node association.
+
+The behavior of the Meta-Scheduling feature is summarized on the following diagram (example for PBS integration):
+
+image::NativeScheduler.png[PBS Integration, , title="PBS Integration"]
```

Comment:
I do not see the image in the build artifact of the PR, to be checked when you merge.

Reply:
I see it:
http://jenkins.activeeon.com/view/Pull%20Requests/job/documentation-pull-requests/jdk=JDK8,label=Linux-non-scheduling-jobs/lastSuccessfulBuild/artifact/build/asciidoc/html5/images/NativeScheduler.png